marklogic-data-hub-central/ui/src/components/explore/graph-vis-explore/graph-vis-explore.tsx

```diff
@@ -254,6 +254,34 @@ const GraphVisExplore: React.FC<Props> = (props) => {
     return expandedNodeData.hasOwnProperty(expandId);
   };
 
+  const createTitleWithPropertiesOnHover = propertiesOnHover => {
```

Comment:
@xnikhil08 I was looking for a way to achieve pass jsx element to the graph-vis node title property but the property expect for an Element of html, not recognize any of react element type because return a differnt type.
In this case there is no direct manipulation of the DOM only use the DOM api to create native html element and pass like parameter.